packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-container.tsx

```diff
@@ -135,10 +133,7 @@ const ProofOfAddressContainer = observer(({ onSubmit }: TProofOfAddressContainer
     );
 
     if (is_loading) return <Loading is_fullscreen={false} className='account__initial-loader' />;
-    if (
-        !allow_document_upload ||
-        (!is_age_verified && !allow_poa_resubmission && document_status === 'none' && is_mx_mlt)
-    )
+    if (!allow_document_upload || (!is_age_verified && !allow_poa_resubmission && document_status === 'none'))
```

Comment:
@matin-deriv `is_mx_mlt` is false and the second part of the condition will be false so we can completely remove it.
```suggestion
    if (!allow_document_upload ) return <NotRequired />;
```